pkg/features/feature.go

```diff
@@ -166,6 +172,25 @@ func InitializeFeatures(featuresClient managementv3.FeatureClient, featureArgs s
 	}
 }
 
+func SetFeature(featuresClient managementv3.FeatureClient, feature *Feature, value bool) error {
+	if featuresClient == nil {
+		return nil
+	}
+
+	featureState, err := featuresClient.Get(feature.name, metav1.GetOptions{})
+	if err != nil {
+		return err
+	}
+
+	featureState.Spec.Value = &[]bool{value}[0]
+	if _, err = featuresClient.Update(featureState); err != nil {
+		return err
+	}
+
+	feature.Set(value)
```

Comment:
There is a handler that will set this automatically. It's important we let the handler do it because it is on the ScaledContext and handles restarting each Rancher node when necessary. Doing this will bypass that restart for whichever node it is used on. Set is something that must occur on every rancher node while feature update only needs to happen once.

Reply:
I see, will change this.